core/embed/bootloader/messages.c

```diff
@@ -680,29 +692,13 @@ int process_msg_FirmwareUpload(uint8_t iface_num, uint32_t msg_size,
 }
 
 secbool bootloader_WipeDevice(void) {
-  static const uint8_t sectors[] = {
-      FLASH_SECTOR_STORAGE_1,
-      FLASH_SECTOR_STORAGE_2,
-      // 3,  // skip because of MPU protection
-      FLASH_SECTOR_FIRMWARE_START,
-      7,
-      8,
-      9,
-      10,
-      FLASH_SECTOR_FIRMWARE_END,
-      FLASH_SECTOR_UNUSED_START,
-      13,
-      14,
```

Comment:
The new code does not wipe these.

Reply:
right, added a WIPE_AREA to precisely define what should be wiped https://github.com/trezor/trezor-firmware/commit/55785b2157d3e49913a1c4140bca60d582d5f63a